packages/utils/typescript/lib/utils/get-config-path.js

```diff
@@ -22,6 +22,8 @@ module.exports = (dir, { filename = DEFAULT_TS_CONFIG_FILENAME, ancestorsLookup
   if (!configFilePath || ancestorsLookup) {
     return configFilePath;
```

Comment:
I guess we'll need to path.resolve here too, no?
Maybe we could split this in two conditions, smth like `no config file => return null/undefined`, `ancestorLookup => path.resolve(configFilePath)`?

Reply:
I don't think that's a necessary step. but i guess something like below would suffice.

```js
if (configFilePath) {
    const configFileAbsolutePath = path.resolve(configFilePath);

    if (ancestorsLookup || configFileAbsolutePath.startsWith(dirAbsolutePath)) {
        return configFileAbsolutePath;
    }
}

return undefined;
```

Reply:
Yep looks good to me :+1:

Reply:
Confirming this patch from @psyirius works :+1:

Reply:
> Hey @psyirius, could you update your branch with main so that we can move forward with this, please?

Oopsie, I stopped checking this pr after checking it for may days for update and no progress.
Glad to hear a fix is landed finally! Patched strapi on production was a nightmare when updating.